frontend/src/pages/add-blog.tsx

```diff
@@ -57,8 +75,7 @@ function AddBlog() {
       toast.error('All fields must be filled out.');
       return false;
     }
-
-    const imageLinkRegex = /\.(jpg|jpeg|png)$/i;
+    const imageLinkRegex = /\.(jpg|jpeg|png|webp)$/i;
     if (!imageLinkRegex.test(formData.imageLink)) {
       toast.error('Image URL must end with .jpg, .jpeg, or .png');
       return false;
```

Comment:
Please add .webp even in the toast message

Reply:
Added